demos/benchmark/lv_demo_benchmark.c

```diff
@@ -948,7 +948,7 @@ static void scene_next_task_cb(lv_timer_t * timer)
 
     if(scenes[scene_act].create_cb) {
         lv_label_set_text_fmt(title, "%"LV_PRId32"/%d: %s%s", scene_act * 2 + (opa_mode ? 1 : 0),
-                              (sizeof(scenes) / sizeof(scene_dsc_t) * 2) - 2,  scenes[scene_act].name, opa_mode ? " + opa" : "");
+                              (dimof(scenes) * 2) - 2,  scenes[scene_act].name, opa_mode ? " + opa" : "");
```

Comment:
Isn't `dimof` only for C++?
I've just  tested it in a few embedded compilers and it wasn't support in all of them.

So I suggest either keeping the original version or adding `LV_DIMOF` to `lv_types.h` (in lack of a better place)

Reply:
it's defined at the begin:
https://github.com/lvgl/lvgl/pull/3114/files#diff-07251bd4728b39cb8b95efcfb4c0ca01b07e6140094d01dbdfa9536c9951e6adR46-R48

Reply:
Oh, I didn't know that :slightly_smiling_face: